oca_custom/__manifest__.py

```diff
@@ -4,10 +4,11 @@
 {
     'name': 'OCA Custom Settings',
     'summary': 'Custom Settings for OCA Instance',
-    'version': '11.0.1.1.0',
+    'version': '12.0.1.0.0',
     'category': 'Custom',
     'author': 'GRAP, '
               'Akretion, '
+              'Tecnativa, '
```

Comment:
We don't deserve co-authorship in this case.

Reply:
Done